styleguide/_themes/derek/scss/snowflakes/events.scss

```diff
@@ -0,0 +1,366 @@
+.events-banner {
+  background-image: url('https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/bananers/backgrounds/solve-banner.jpg');
+}
+
+// Main Events Page.
+.eventsOverview {
+  @include make-container;
+}
+
+.eventsOverview-pagination {
+  list-style-type: none;
+  margin: 0;
+  padding: 2rem 0 0;
+  text-align: center;
+}
+
+.eventsOverview-pagination-item {
+  display: inline-block;
+  list-style-type: none;
+}
+
+.eventsOverview-pagination-itemPage {
+  background: $white;
+  border: 1px solid $gray-light;
+  font-size: 1rem;
+  padding: .5rem;
+  text-decoration: none;
+
+  &:hover {
+    background: $gray-lightest;
+    color: $black;
```

Comment:
should be gray-darkest. We're not using black for any text. only icons and what not.